DoctrineBundle.php

```diff
@@ -119,7 +119,7 @@ public function shutdown()
         // Clear all entity managers to clear references to entities for GC
         if ($this->container->hasParameter('doctrine.entity_managers')) {
             foreach ($this->container->getParameter('doctrine.entity_managers') as $id) {
-                if (!$this->container instanceof IntrospectableContainerInterface || $this->container->initialized($id)) {
+                if (!method_exists($this->container, 'initialized') || $this->container->initialized($id)) {
```

Comment:
`IntrospectableContainerInterface` use can be removed

Reply:
done